corehq/motech/dhis2/templates/dhis2/dhis2_entity_config.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'hqwebapp/two_column.html' %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+{% load compress %}
+
+{% block title %}Edit Config : DHIS2 Tracked Entities :: {% endblock %}
+
+{% block js %}{{ block.super }}
```

Comment:
You should be able to remove this block, requirejs pages shouldn't have script tags.

Reply:
You are right ... but there is something I'm not sure about: "mode-json.js", "ext-searchbox.js" and "mode-xml.js" each throw "ReferenceError: ace is not defined" ... but the page works fine: JSON formatting works, and the search box works. Are those errors expected?

Reply:
I'm pretty sure I've seen that in prod somewhere else. So it's not expected exactly but I think not specific to this page.